CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ ENDIF()
 # I really wish these were regular variables
 # but scoping issues can make it difficult
 GLOBAL_SET(KOKKOS_COMPILE_OPTIONS)
-GLOBAL_SET(KOKKOS_LINK_OPTIONS -DKOKKOS_DEPENDENCE)
```

Comment:
`-DKOKKOS_DEPENDENCE` is passed to the linker so that `kokkos_launch_compiler` knows whether or not the target being linked is a kokkos target or not. So it cannot be removed without breaking the launch-compiler additions. You should either use `/DKOKKOS_DEPENDENCE` for windows or just disable it _only_ on windows